lib/v2/sicnu/router.js

```diff
@@ -0,0 +1,3 @@
+module.exports = function (router) {
+    router.get('/www/:category', require('./www'));
```

Comment:
https://github.com/DIYgod/RSSHub/blob/92d9c57cbe02e9deb6298642fbeba1616573a01a/lib/v2/sicnu/www.js#L5
```suggestion
    router.get('/www/:category?', require('./www'));
```